### ISSUE TITLE
Chrome doesn't close during 'invalid url' error

### ISSUE THREAD

**Issue by ramlmn** [GoogleChrome/lighthouse#1002]
Lighthouse doesn't kill any instances of chrome when an error or exception occurs during the audit or at the start of the audit like when an error is used to reject the Promise(`lighthouse-core/runner.js:46`) when the url passed is invalid(like: `file:///D:/demo.html`) lighthouse exits without closing chrome instance:
```
$ lighthouse file:///D:/demo.html
  Lighthouse CLI Launching Chrome... +0ms
  ChromeLauncher Waiting for browser. +31ms
  ChromeLauncher Waiting for browser... +0ms
  ChromeLauncher Waiting for browser...✓ +513ms
Runtime error encountered: Error: The url provided should have a proper protocol and hostname.
    at Function.run (C:\Users\ramlmn\AppData\Roaming\npm\node_modules\lighthouse\lighthouse-core\runner.js:45:19)
    at Promise (C:\Users\ramlmn\AppData\Roaming\npm\node_modules\lighthouse\lighthouse-core\index.js:64:20)
    at module.exports (C:\Users\ramlmn\AppData\Roaming\npm\node_modules\lighthouse\lighthouse-core\index.js:47:10)
    at lighthouseRun (C:\Users\ramlmn\AppData\Roaming\npm\node_modules\lighthouse\lighthouse-cli\bin.js:191:12)
    at launcher.isDebuggerReady.catch.then (C:\Users\ramlmn\AppData\Roaming\npm\node_modules\lighthouse\lighthouse-cli\bin.js:180:25)
Error: The url provided should have a proper protocol and hostname.
    at Function.run (C:\Users\ramlmn\AppData\Roaming\npm\node_modules\lighthouse\lighthouse-core\runner.js:45:19)
    at Promise (C:\Users\ramlmn\AppData\Roaming\npm\node_modules\lighthouse\lighthouse-core\index.js:64:20)
    at module.exports (C:\Users\ramlmn\AppData\Roaming\npm\node_modules\lighthouse\lighthouse-core\index.js:47:10)
    at lighthouseRun (C:\Users\ramlmn\AppData\Roaming\npm\node_modules\lighthouse\lighthouse-cli\bin.js:191:12)
    at launcher.isDebuggerReady.catch.then (C:\Users\ramlmn\AppData\Roaming\npm\node_modules\lighthouse\lighthouse-cli\bin.js:180:25)
```

I don't think that this is a suggested behaviour. It can be fixed by catching the Promise rejection in `lighthouse-cli/bin.js:181` by adding:
``` js
.catch((error) => {
  console.error(error);
  launcher.kill();
});
```

Is this behaviour is acceptable then you can close this issue or else can I make a PR for this please?

**Comment by ebidel** [comment]
PR is welcome :)

**Comment by brendankenny** [comment]
fixed in #1141
